backend/eventloop/shared_resource_loop/sharedresourceloop_managedenv.go

```diff
@@ -927,36 +967,6 @@ func sanityTestCredentials(clusterCreds db.ClusterCredentials) (*rest.Config, bo
 	return configParam, true, nil
 }
 
-// verifyClusterCredentials returns true if we were able to successfully connect with the credentials, false otherwise.
```

Comment:
This old function is no longer used for verifying cluster credentials. The problem with this function was that it assumed the user could see `ServiceAccounts` in the `kube-system` namespace, which is not true in many use cases.

Instead we just check to see if the ServiceAccount can view Namespace (there is a new function for this).